includes/woopay/class-woopay-session.php

```diff
@@ -493,10 +486,6 @@ private static function get_woopay_verified_email_address() {
 	 * @return bool True if request is a Store API request, false otherwise.
 	 */
 	private static function is_store_api_request(): bool {
-		if ( ! defined( 'REST_REQUEST' ) || ! REST_REQUEST ) {
-			return false;
-		}
```

Comment:
Is removing this necessary? I guess maybe some of the checks are happening early enough that `REST_REQUEST` hasn't been defined yet, but it should only be possible to interact with the Store API through REST requests so I'm a bit surprised to see that we need to remove this condition 🤔

Reply:
> I guess maybe some of the checks are happening early enough that REST_REQUEST hasn't been defined yet

From my discussions with @ricardo that's exactly what was happening, the constant wasn't defined yet. I agree, the condition itself would make sense to keep, but again from my understanding the constant not being defined yet made some checks fail here and then prevented getting the correct shopper set to get their currency.